project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.1.6
+sbt.version=1.2.1
```

Comment:
Updated to 1.2.1 and adapted builds to explicitly include `ScriptedPlugin` when needed.